user_manual/groupware/calendar.rst

```diff
@@ -211,7 +218,20 @@ An event may be set as "recurring", so that it can happen every day, week, month
 You can also tell when the recurrence ends.
 
 .. figure:: images/calendar_event_repeat.png
+              :scale: 80%
+
+.. _calendar-trash-bin:
+
+Trash bin
+~~~~~~~~~
+
+If you delete events, tasks or a calendar in Calendar, your data won't be gone yet. Instead, those items will be collected in a *trash bin*. This offers you to undo a deletion. After a period of 30 days those items will be deleted permanently. You can also permanently delete earlier if you wish.
```

Comment:
You can delete them from other clients as well.

Reply:
how? delete from other clients will move the event to the trash bin. the trash bin is not accessible for other clients

Reply:
I mean the events are placed into the trashbin even if the events are deleted from other clients, but we need to make it clear the trashbin is accessible only through the Calendar app.